### PR TITLE
Update forwarding.rst

### PR DESCRIPTION
**Pull request by 94noni**
some code clean, (and add comment to which file we are talking of)

### CODE REVIEW

File: controller/forwarding.rst [hunk 1]
```diff
@@ -13,8 +13,10 @@ Instead of redirecting the user's browser, this makes an "internal" sub-request
 and calls the defined controller. The ``forward()`` method returns the
 :class:`Symfony\\Component\\HttpFoundation\\Response` object that is returned
 from *that* controller::
+    
+    // src/Controller/OneController.php
```
Comment (OskarStark) [inline controller/forwarding.rst]: No need for the path here

File: controller/forwarding.rst [hunk 2]
```diff
@@ -29,7 +31,9 @@ from *that* controller::
 The array passed to the method becomes the arguments for the resulting controller.
 The target controller method might look something like this::
 
-    public function fancy($name, $color)
+    // src/Controller/OtherController.php
```
Comment (OskarStark) [inline controller/forwarding.rst]: No need for the path here

**Comment by 94noni** [comment]
@OskarStark i've reverted adding the path, even if at first it was the main topic of the PR :)

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @94noni.

**Pull request merged**